src/components/NoteExpand/NoteExpandContainer.tsx

```diff
@@ -3,7 +3,7 @@ import { useSelector } from 'react-redux';
 import { useParams } from 'react-router-dom';
 import { AppStateType } from 'redux/redux-store';
 
-import GoToHome from 'components/Common/GoHome/GoHome';
+import GoToHome from 'components/Common/GoBack/GoBack';
```

Comment:
change to `GoBack`

Reply:
i thinked what reafactoring all components will happen by itself... ready